ansible/roles/operator-pipeline/templates/openshift/tasks/git-clone.yml

```diff
@@ -211,3 +211,4 @@ spec:
         fi
         printf "%s" "${RESULT_SHA}" > "$(results.commit.path)"
         printf "%s" "${PARAM_URL}" > "$(results.url.path)"
+        chmod -R 777 "${CHECKOUT_DIR}"
```

Comment:
Does this dirty the git status? Please confirm if it works with digest pinning enabled. We need to ensure it won't produce a commit with every file permission changed.